android/KMEA/app/src/main/java/com/tavultesoft/kmea/data/CloudRepository.java

```diff
@@ -61,6 +62,11 @@ private CloudRepository() {
     lastLoad = null;
   }
 
+  // For local and PR test builds, force check of keyboard updates
+  private static boolean isTestBuild() {
```

Comment:
```suggestion
  private static boolean isLocalOrTestBuild() {
```

Given you are using the same pattern also in other source files, perhaps you should move this to VersionUtils.java and update references accordingly?

Reply:
Moving to `VersionUtils.isLocalOrTestBuild()`